packages/devtools_app/integration_test/test/live_connection/eval_and_browse_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Do not delete these arguments. They are parsed parsed by test runner.
+// test-argument:experimentsOn=true
```

Comment:
nit: use `experiments-on` or `experiments_on` to better match standard argument formatting. We could also just use `experiments` to simplify

Reply:
This mean I will need to introduce field for enum of the arguments instead of just using their name.

May be it is ok to have different standard for different type of arguments.

Reply:
> This mean I will need to introduce an additional field for the enum of the arguments instead of just using their name.

Can you explain what you mean by this? I'm not following. I'm just suggesting a trivial change from the key "experimentsOn" to "experiments" or "experiments-on"

Reply:
after looking at enum, I see what you mean. leaving as is sgtm.